### ISSUE TITLE
[Question] Regarding the issue of one-shot books within series books directory.

### ISSUE THREAD

**Issue by yatounoneko** [gotson/komga#1452]
### Steps to reproduce

file directory：e.g.

/data/books/Comics
├── Space Adventures
│   ├── Space_Adventures_001.cbz
│   ├── Space_Adventures_002.cbz
│   ├── Space_Adventures_003.cbz
├── Super Duck
│   └── Super_Duck_001.cbz
├── abc.zip
├── cbd.zip
├── efg.zip
├── hig.rar
└──  klm.cbr

### Expected behavior

Hi, my file directory is roughly like this. (Due to BT reasons, I cannot create a separate directory for one-shot.)
One-shot book can establish a comic series (with only one episode in the series) if there is have a directory.

### Actual behavior

Because the directory contains a series of comics and one-shot books, so these one-short files (e.g., efg.zip, hig.rar…) have all turned into a series of books.

- 3 regular series:
  - Space Adventures
  - Super Duck
  - Comics (One-Shots series)
     1. abc.zip
     2. cbd.zip
     3. efg.zip
     4. hig.rar
     5. klm.cbr

### Komga version

1.10.3

### Operating system

Windows server 2019

### Installation method

jar

### Other details

Is there a way to identify a one-shot book without creating a one-shot directory? Would it work if I fill in the entire directory path(including series book) into the one-shot directory parameter?

Thanks everyone for this issue.

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
> (Due to BT reasons, I cannot create a separate directory for one-shot.)

that's a 'you' problem. Use hardlinks, or docker mounts. Komga needs a folder below the root, even for oneshots.

**Comment by yatounoneko** [comment]
Thank you for taking the time to reply my problem and offer suggestions; I will try the method you mentioned.

I would like to explain my usage situation and offer some suggestions: 

Currently, after creating a new library, if a one-shot directory is not set in the library configuration, a series named after the library is automatically generated. 

I can copy the file paths from these series one by one to create hard links to a separate directory. However, my directory is automatically updated and downloaded via a BT client using RSS, which cannot automatically categorize one-shot books. Therefore, every time a new seed is downloaded, I need to update the hard link directory one by one.

In summary, I wonder if there could be an option to classify all single files under the designated library’s setting directory as one-shots(_Or will it be available in the future_)?

Thank you for developing Komga, which has made reading comics more enjoyable and convenient for everyone.
